README.md

```diff
@@ -45,7 +45,7 @@ $ sudo systemctl start mosquitto
 ### Build and Run test
 ```
 # cd $NNST_EDGE_ROOT
-$ cmake -B build -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_INSTALL_LIBDIR=lib -DENABLE_TEST=ON -DMQTT_SUPPORT=ON
+$ cmake -B build -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_INSTALL_LIBDIR=/lib -DENABLE_TEST=ON -DMQTT_SUPPORT=ON
```

Comment:
Nope. We don't install our .so files at /lib. I think the CMakeLists.txt is not handling install direcctories properly for creating .pc files. It should depend on PREFIX. (i.e., $PREFIX/$LIBDIR)

Reply:
@myungjoo Thanks for your comment.
I agree your comment.  It seems that the LIBDIR path is not used properly in the CMakeLists.txt file.
I will close this PR and fix the problem, and create a new PR.

thank you!! :+1: